fs/procfs/fs_procfsmeminfo.c

```diff
@@ -382,6 +391,88 @@ static ssize_t meminfo_read(FAR struct file *filep, FAR char *buffer,
   return totalsize;
 }
 
+/****************************************************************************
+ * Name: memdump_read
```

Comment:
Please add Description

Reply:
None of the xxx_read/xxx_write was description in fs/procfs. Do i need to add description for all xxx_read/xxx_write?

Reply:
ping @acassis do you think it's acceptable to follow the old style?